packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -382,7 +382,7 @@ export class FluidDevtools implements IFluidDevtools {
 		return {
 			telemetry: this.logger !== undefined,
 			// Completed but disabled until we finish the story for unsampled telemetry that powers this feature
-			opLatencyTelemetry: false,
+			opLatencyTelemetry: true,
```

Comment:
Don't forget to revert this boolean change prior to merging

Reply:
Weren't we meant to save enable this once it was operational? Or should I update the comment to say that it's a feature flag that's disabled by default?

Reply:
I think making it true by default makes sense now that it exists. I might even argue for removing the line altogether, particularly if we go with the suggestion of adding the instructions on how to disable sampling in the same OpLatencyView, so people trying to use the feature have a natural way of discovering how to use it.